app/[locale]/(interactive)/feedback/page.tsx

```diff
@@ -27,7 +29,7 @@ const Page = (props: Props) => {
 
     if (submissionStatus === "SUBMITTING") return;
     if (isEmpty) {
-      router.push("/end");
+      router.push(`/${locale}/end`);
```

Comment:
use `import { useRouter } from "@/lib/navigation` kub it automatically handles the locale for you so you don't need to specifically specify locale